From 09510adf07b9571e639873663ddf1267f12cfb81 Mon Sep 17 00:00:00 2001
From: Your Name <1446775+jacaru@users.noreply.github.com>
Date: Sat, 25 Mar 2023 19:33:39 +0100
Subject: [PATCH 3/5] Use relative href for app icons

While this could be achieved with the BASE URL setting, that is not a
one stop shop to be able to run this app on a sub-path. I find that
direcly served assets just work better with a relative path.
---
 src/pages/index.jsx | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/src/pages/index.jsx b/src/pages/index.jsx
index a4bd0e96..cfd8b692 100644
--- a/src/pages/index.jsx
+++ b/src/pages/index.jsx
@@ -241,10 +241,10 @@ function Home({ initialSettings }) {
           </>
         ) : (
           <>
-            <link rel="apple-touch-icon" sizes="180x180" href="/apple-touch-icon.png?v=4" />
-            <link rel="shortcut icon" href="/homepage.ico" />
-            <link rel="icon" type="image/png" sizes="32x32" href="/favicon-32x32.png?v=4" />
-            <link rel="icon" type="image/png" sizes="16x16" href="/favicon-16x16.png?v=4" />
+            <link rel="apple-touch-icon" sizes="180x180" href="apple-touch-icon.png?v=4" />
+            <link rel="shortcut icon" href="homepage.ico" />
+            <link rel="icon" type="image/png" sizes="32x32" href="favicon-32x32.png?v=4" />
+            <link rel="icon" type="image/png" sizes="16x16" href="favicon-16x16.png?v=4" />
           </>
         )}
         <meta
-- 
2.30.2

